docs/checkers/index.md

```diff
@@ -422,6 +422,26 @@ else:
     number_category = 'non-negative'
 ```
 
+(C0209)=
+
+### Consider using f string (C0209) [](#C0209)
+
+This error occurs when a string is formatted with `%` or `format()`.
+
+```python
```

Comment:
You can use the `literalinclude` directive to include your example file, so that you don't need to repeat the contents again here. (See how we did this with other examples.)